pingo/__init__.py

```diff
@@ -1,32 +1,38 @@
+from __future__ import absolute_import
+
 # api
-from board import ANALOG  # noqa
-from board import IN  # noqa
-from board import OUT  # noqa
-from board import PWM  # noqa
-from board import HIGH  # noqa
-from board import LOW  # noqa
-from board import ModeNotSuported  # noqa
-from board import WrongPinMode  # noqa
-from board import PwmOutputCapable  # noqa
-from board import AnalogInputCapable  # noqa
-from board import Board  # noqa
-from board import PwmPin  # noqa
-from board import AnalogPin  # noqa
-from board import DigitalPin  # noqa
-from board import GroundPin  # noqa
-from board import Pin  # noqa
-from board import VccPin  # noqa
-import parts  # noqa
+from .board import State  # noqa
+from .board import Mode  # noqa
+# from .board import ANALOG  # noqa
```

Comment:
If you created a mode, there is no need to keep this lines.
You could delete the commented lines.